lesson_07/homeworks_07.py

```diff
@@ -31,27 +31,39 @@ def multiplication_table(number):
 # task 2
 """  Написати функцію, яка обчислює суму двох чисел.
 """
-
+def summa (a,b):
+    return a+b
+print("Task 3 - returned sum of a + b = ", summa(2,3))
 
 # task 3
 """  Написати функцію, яка розрахує середнє арифметичне списку чисел.
 """
-
+numbers = [1, 2, 3]
+def arithmetic_mean (a):
+    return sum(a) / len(a)
```

Comment:
```suggestion
def arithmetic_mean(array:list):
    """ Doc string """
    return sum(array) / len(array)
```
звертайте увагу на імена
робіть зручність для користування - документація важлива.